Templates/DLS.ini

```diff
@@ -11,15 +11,15 @@ DisabledControls=80,86,27
 
 # Extra patch
 ; Enables or disables the extra patch (Default = false)
-ExtraPatch=true
+ExtraPatch=false
 
 # Developer Mode
 ; Enables or disables the developer mode (Default = false)
-DevMode=true
+DevMode=false
```

Comment:
`DevMove` is currently enabled prior to release in purpose as all that should have access to it are developers.